Readme.txt

```diff
@@ -39,144 +40,188 @@ CONFIGURATION
     1) Select the desired contest using the Contest drop-down list.
     2) Enter the Contest Exchange in the Exchange field;
        any error messages will be displayed in the status area.
-       * Cut Numbers in sending exchanges coming soon.
+       (please keep in mind that the IARU HST Competition mode might ignore
+        some settings so as to behave like the original program.)
 
-    More contest information is at the bottom of this document.
+    Specific contest information is at very bottom of this document.
 
   Station
-    Call - enter your contest callsign here.
+    Call - enter your contest call sign here.
     QSK - simulates the semi-duplex operation of the radio. Enable it if your
-      physical radio supports QSK. If it doesn't, enable QSK anyway to see
-      what you are missing.
+          physical radio supports QSK. If it doesn't, enable QSK anyway to see
+          what you are missing.
     CW Speed - select the CW speed, in WPM (PARIS system) that matches your
-      skills. The calling stations will call you at about the same speed.
+               skills. The calling stations will call you at about the same speed.
+               There are more speed controls under the settings menu.
     CW Pitch - pitch in Hz.
-    RX Bandwidth - the receiver bandwidth, in Hz.
-    Audio Recording Enabled - when this menu option is checked, MR saves
-      the audio in the MorseRunner.wav file. If this file already
-      exists, MR overwrites it.
+    RX Bandwidth - there are several ways to adjust this. See the key
+                   the key assignments below.
```

Comment:
"the key" is repeated twice (lines 57 and 58)

Reply:
Thanks!